README.md

```diff
@@ -1,739 +1,18 @@
-
-![Movie landed in Plex](img/busan_plex.png)
-
-#### Movie discovering
-
-I like the discovering feature. When clicking on `Add Movies` you can select `Discover New Movies`, then browse through a list of TheMovieDB recommended or popular movies.
-
-![Movie landed in Plex](img/radarr_recommendations.png)
-
-On the rightmost tab, you'll also see that you can setup Lists of movies. What if you could have in there a list of the 250 greatest movies of all time and just one-click download the ones you want?
-
-This can be set up in `Settings/Lists`. I activated the following lists:
-
-- StevenLu: that's an [interesting project](https://github.com/sjlu/popular-movies) that tries to determine by certain heuristics the current popular movies.
-- IMDB TOP 250 movies of all times from Radarr Lists presets
-- Trakt Lists Trending and Popular movies
-
-I disabled automatic sync for these lists: I want them to show when I add a new movie, but I don't want every item of these lists to be automatically synced with my movie library.
-
-## Manage it all from your mobile
-
-On Android, I'm using [nzb360](http://nzb360.com) to manage NZBGet, Deluge, Sonarr and Radarr.
-It's a beautiful and well-thinked app. Easy to get a look at upcoming tv shows releases (eg. "when will the next f**cking Game of Thrones episode be released?").
-
-![NZB360](img/nzb360.png)
-
-The free version does not allow you to add new shows. Consider switching to the paid version (6$) and support the developer.
-
-## Going Further
-
-Some stuff worth looking at that I do not use at the moment:
-
-- [NZBHydra](https://github.com/theotherp/nzbhydra): meta search for NZB indexers (like [Jackett](https://github.com/Jackett/Jackett) does for torrents). Could simplify and centralise nzb indexers configuration at a single place.
-- [Organizr](https://github.com/causefx/Organizr): Embed all these services in a single webpage with tab-based navigation
-- [Plex sharing features](https://www.plex.tv/features/#feat-modal)
-- [Headphones](https://github.com/rembo10/headphones): Automated music download. Like Sonarr but for music albums. I've been using it for a while, but it did not give me satisfying results. I also tend to rely entirely on a Spotify premium account to manage my music collection now.
-- [Mylar](https://github.com/evilhero/mylar): like Sonarr, but for comic books.
-- [Ombi](http://www.ombi.io/): Web UI to give your shared Plex instance users the ability to request new content
-- [PlexPy](https://github.com/JonnyWong16/plexpy): Monitoring interface for Plex. Useful is you share your Plex server to multiple users.
-- Radarr lists automated downloads, to fetch best movies automatically. [Rotten Tomatoes certified movies](https://www.rottentomatoes.com/browse/cf-in-theaters/) would be a nice list to parse and get automatically.
+Visit the [docs](https://johnpyp.github.io/htpc-download-box)
```

Comment:
At least for me, the bigger reason for using docsify is it makes editing/adding a lot easier since you aren't editing a massive 700 line page.

Maintaining both a Github readme and Docsify seems a bit difficult, and I think navigating Docsify is generally going to be a lot easier for people who are following it, as they look back for specific configuration details or whatever.

The main README is very sparse right now, yeah. I wasn't really sure what to put to be honest. I've never been very good at the descriptive/selling point kind of summaries.